incompatibility/incomp-scanning_test.go

```diff
@@ -67,9 +68,10 @@ func TestIncompatibilityExistence(t *testing.T) {
 	var existenceTest = []struct {
 		path string
 	}{
-		{"../../examples/petstore/petstore.yaml"},
-		{"../oas-examples/petstore.json"},
-		{"../../examples/bookstore/bookstore.yaml"},
+		{"../examples/petstore/petstore.yaml"},
```

Comment:
I don't know that I fully follow what you are testing with this test?  It seems that you intend to test at least one incompatibility is found and that you can map that to a node in the yaml.  

But you aren't verifying any specific incompatibilities are being found which is what the method names seems to imply to me.